admin_guide/managing_networking.adoc

```diff
@@ -1246,6 +1246,67 @@ IP address. If `egressIPs` is set on a `NetNamespace`, but there is no node
 hosting that egress IP, then egress traffic from the namespace will be dropped.
 endif::openshift-origin,openshift-enterprise[]
 
+[[admin-guide-automatic-egress-ip]]
+=== Enabling Automatic Egress IPs
+
+Similar to xref:enabling-static-ips-for-external-project-traffic[Enabling Static
+IPs for External Project Traffic], as a cluster administrator you can assign
+egress IPs to namespaces by setting the `egressIPs` field to the `NetNamespace`
+resource. With fully-automatic egress IPs, you can set the `egressCIDRs` field
```

Comment:
IBM Style says not to use a hyphen after an adverb or adjective that ends in -ly, so fully-automatic should be fully automatic